index.ts

```diff
@@ -238,25 +267,25 @@ function ensureTypeScriptInstance(loaderOptions: LoaderOptions, loader: any): {
         compilerOptions.module = 0 /* None */;
         libFileName = 'lib.es6.d.ts';
     }
-    
+
     if (loaderOptions.transpileOnly) {
         // quick return for transpiling
         // we do need to check for any issues with TS options though
         var program = compiler.createProgram([], compilerOptions),
-            diagnostics = program.getOptionsDiagnostics 
-                ? program.getOptionsDiagnostics() 
-                : program.getCompilerOptionsDiagnostics();
+            diagnostics = program.getOptionsDiagnostics
+                ? program.getOptionsDiagnostics()
+                : languageService.getCompilerOptionsDiagnostics();
```

Comment:
Ah, you're right. I think `getCompilerOptionsDiagnostics` must have been there at some point in the nightly or something. Looks like that branch can just be removed and this can just be `diagnostics = program.getOptionsDiagnostics()`;